NuRadioReco/modules/io/coreas/simulationSelector.py

```diff
@@ -81,8 +81,9 @@ def run(self, evt, sim_station, det, frequency_window = [100*units.MHz,500*units
             noise_std = np.std(noise_region)
 
             noise += n_std * noise_std
-            
-            mask = (freq >= frequency_window.min())  & (freq <= frequency_window.max())
+
+            mask = np.array((freq >= np.min(frequency_window))  & (freq <= np.max(frequency_window)))
```

Comment:
I don't think the np.array command is really necessary here

Reply:
It is if you give it a list for the frequency window as the documentation suggests.